docs/components.md

```diff
@@ -73,6 +73,13 @@ var ClockModel = BaseModel.extend({
 });
 ```
 
+Shown below is how the `change` event on `time_component` is exposed.
+
+var TimeModel = BaseModel.extend({
+  exposedEvents: ['change'],
```

Comment:
Cold we add an example on what this looks like if you expose a specific change event, say `change:selected` as well as a custom event like `foo`?